docs/Users_Guide/roc_diagram.rst

```diff
@@ -51,27 +52,103 @@ METplus Configuration
+with the full path `$WORKING_DIR/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+For example:
+
+`stat_input: /username/working_dir/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+`plot_filename: /username/working_dir/output_plots/roc_diagram_custom.png`
+
+where $WORKING_DIR was replaced with the actual name of the directory.  In this example, the
+/username/working_dir/output_plots directory exists and has the appropriate read and write permissions.
+
+
+If you wish to save the intermediate `.points1` file (used by METviewer but also useful for debugging), set the following configuration values:
+
+`dump_points_1 setting: 'True'`
 
-If the user
-wishes to use all the default settings defined in the roc_diagram_defaults.yaml
-file, an empty custom configuration file (minimal_roc_diagram_defaults.yaml)
-can be specified instead:
+And add this line (anywhere in the file) if it doesn't already exist, to set the optional points_path:
 
-.. literalinclude:: ../../metplotpy/plots/roc_diagram/minimal_roc_diagram.yaml
+`points_path: '/dir_to_save_points1_file'`
 
 
+Replace the `'/dir_to_save_points1_file'` to the directory where you wish to save the file.
+
+
+**Using defaults**
+
+If you wish to use the **default** settings defined in the roc_diagram_defaults.yaml
+file, specify a minimal custom configuration file (minimal_roc_diagram_defaults.yaml), which consists of only a comment
+block, but can be any empty file:
+
+.. literalinclude:: ../../test/roc_diagram/minimal_roc_diagram.yaml
+
+copy this file to your working directory:
+
+``cp $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/minimal_roc_diagram.yaml $WORKING_DIR/minimal_roc_diagram.yaml``
+
+Add the `stat_input` (input data) and `plot_filename` (output file/plot) settings to the $WORKING_DIR/minimal_roc_diagram.yaml
+file (anywhere below the comment block). This explicitly indicates where the
+sample data and custom configuration files are located.  Set the `stat_input` to
+`$METPLOTPY_SOURCE/METplotpy/test/roc_diagram/plot_20200507_074426.data` and set the
+`plot_filename` to  $WORKING_DIR/output_plots/roc_diagram_default.png:
+
+`stat_input: $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+`plot_filename: $WORKING_DIR/output_plots/roc_diagram_default.png`
+
+Where `$WORKING_DIR` is the working directory where you are saving all your custom
+configuration files. **NOTE**: You may specify the `plot_filename` (output directory) to a directory other than the
+$WORKING_DIR/output_plots, as long as it is an existing directory where you have read and write permissions.
+
+If you wish to save the intermediate `.points1` file (used by METviewer and useful for debugging), add the following lines to your minimal_roc_diagram.yaml
+file (anywhere below the comment block):
+
+`dump_points_1: 'True'`
+
+`points_path: '/dir_to_save_points1_file'`
+
+
+Replace the `'/dir_to_save_points1_file'` to the directory where you wish to save the `.points1` file.
```

Comment:
```suggestion
Replace the `'/dir_to_save_points1_file'` to the directory where you wish to save the `.points1` file.  If points_path is commented out (indicated by a '#' symbol in front of it), remove the '#' symbol to uncomment the points_path so that it will be used by the code.
```